index.d.ts

```diff
@@ -288,13 +288,25 @@ declare namespace execa {
 		Similar to [`childProcess.kill()`](https://nodejs.org/api/child_process.html#child_process_subprocess_kill_signal). This is preferred when cancelling the child process execution as the error is more descriptive and [`childProcessResult.isCanceled`](#iscanceled) is set to `true`.
 		*/
 		cancel(): void;
+
+		/**
+		 * Kills the subprocess.
+		 *
+		 * Causes the promise to reject an error with a `.isKilled = true` property.
+		 */
```

Comment:
This comment should be identical to the `README` description (including Markdown links).

Reply:
Done: 50e52ad1a3bb510754557ad96bc0150beacce2ec